sql/core/src/test/resources/sql-tests/inputs/subquery/scalar-subquery/scalar-subquery-predicate.sql

```diff
@@ -279,3 +279,4 @@ HAVING   max(t1b) <= (SELECT   max(t2b)
                       FROM     t2
                       WHERE    t2c = t1c
                       GROUP BY t2c);
+
```

Comment:
Ur, please revert this, too. And we need to update PR description. Especially, the following. We had better keep narrow context in this PR.
```
This is to help SQLQueryTestSuite anonymize these expression ids.
```